# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/intersection-observer/resources/nested-cross-origin-grand-child-iframe.html
# Commit: b8ad0267e88a
# Full Hash: b8ad0267e88abff0a54a6f0f1b5de2ad3a018460
# Author: Hiroyuki Ikezoe <hikezoe.birchill@mozilla.com>
# Date: 2020-02-11 15:57:30
# Regressor Bug: 1599795
# File Overlap Count: 1
# Description:
#   Bug 1599795 - Make IntersectionObserver work in fission world. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D61938
# ==============================================================================

diff -r 3985a60a75b3 -r b8ad0267e88a testing/web-platform/tests/intersection-observer/resources/nested-cross-origin-grand-child-iframe.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/intersection-observer/resources/nested-cross-origin-grand-child-iframe.html	Tue Feb 11 03:59:14 2020 +0000
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<div id="target" style="height: 100px; background-color: green;"></div>
+<script>
+const observer = new IntersectionObserver(records => {
+  records.forEach(record => {
+    if (record.isIntersecting) {
+      window.top.postMessage(record.isIntersecting, "*");
+    }
+  });
+}, {});
+observer.observe(target);
+window.addEventListener("load", () => {
+  window.top.postMessage("ready", "*");
+});
+</script>
